Sources/Build/describe().swift

```diff
@@ -128,6 +128,11 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
     #if os(Linux)
          args += ["-fPIC"]
     #endif
+    #if os(OSX)
+        if let sysroot = Toolchain.sysroot {
```

Comment:
@shmuelk What I would expect to happen is that it would exist, but be nil and not used. The compiler sys root options are supported on all platforms though, and it could conceivably be used one day (e.g., to support cross compilation).